change/@fluentui-react-avatar-e8c58b22-266b-4609-9169-1a26cdef500c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 163 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27292&commitId=42c97fa2102461b226fd0ca57a3b04787ed3fa5a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1740 screenshots unchanged, 0 screenshots with different dimensions and 163 screenshots with visible difference.


  <details><summary><b>unknown</b> 163 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|3| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|9| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|34| Changed|
|Avatar Converged.basic.normal.chromium.png|6| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|83| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|47| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|70| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|43| Changed|
|Persona Converged.basic - Dark Mode.normal.chromium.png|1| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|5| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|2| Changed|
|Persona Converged.basic.normal.chromium.png|3| Changed|
|Persona Converged.textWrap.normal.chromium.png|3| Changed|
|Table flex - subtle selection.rest.chromium.png|9| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|6| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|19| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|33| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|8| Changed|
|Table layout flex - cell actions.always visible.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|9| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|6| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|19| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|33| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|9| Changed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|8| Changed|
|Table layout flex - cell actions.default.chromium.png|9| Changed|
|Table layout flex - cell actions.default.hover row.chromium.png|9| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|9| Changed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|9| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|6| Changed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|6| Changed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|6| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|19| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|19| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|19| Changed|
|Table layout flex - headers.sortable.chromium.png|9| Changed|
|Table layout flex - headers.sortable.hover header.chromium.png|9| Changed|
|Table layout flex - headers.sortable.press header.chromium.png|9| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|9| Changed|
|Table layout flex - truncate.false.chromium.png|9| Changed|
|Table layout flex - truncate.true.chromium.png|9| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|8| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|111| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|7| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|11| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|5| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|76| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|8| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|65| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|8| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|10| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|48| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|9| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|11| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|6| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|19| Changed|
|Table layout flex.multiselect - RTL.chromium.png|9| Changed|
|Table layout flex.multiselect.chromium.png|9| Changed|
|Table layout flex.primary cell.chromium.png|9| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|19| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|42| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|10| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|33| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|10| Changed|
|Table layout flex.single select (checked).chromium.png|9| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select - High Contrast.chromium.png|19| Changed|
|Table layout flex.single select - RTL.chromium.png|9| Changed|
|Table layout flex.single select.chromium.png|9| Changed|
|Table layout flex.size - extra small.chromium.png|7| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|6| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|19| Changed|
|Table layout flex.size - medium - RTL.chromium.png|9| Changed|
|Table layout flex.size - medium.chromium.png|9| Changed|
|Table layout flex.size - small.chromium.png|7| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|6| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|19| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|33| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|9| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|8| Changed|
|Table layout table - cell actions.always visible.chromium.png|9| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|9| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|6| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|19| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|33| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|9| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|8| Changed|
|Table layout table - cell actions.default.chromium.png|9| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|9| Changed|
|Table layout table - cell actions.in header cell.chromium.png|9| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|9| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|6| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|6| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|6| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|19| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|19| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|19| Changed|
|Table layout table - headers.sortable.chromium.png|9| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|9| Changed|
|Table layout table - headers.sortable.press header.chromium.png|9| Changed|
|Table layout table - truncate.default (disabled).chromium.png|9| Changed|
|Table layout table - truncate.false.chromium.png|9| Changed|
|Table layout table - truncate.true.chromium.png|9| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|8| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|111| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|7| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|11| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|5| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|76| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|8| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|65| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|8| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|10| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|48| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|9| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|11| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|6| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|19| Changed|
|Table layout table.multiselect - RTL.chromium.png|9| Changed|
|Table layout table.multiselect.chromium.png|9| Changed|
|Table layout table.primary cell.chromium.png|9| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|19| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|42| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - brand.chromium.png|10| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|33| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|10| Changed|
|Table layout table.single select (checked).chromium.png|9| Changed|
|Table layout table.single select - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select - High Contrast.chromium.png|19| Changed|
|Table layout table.single select - RTL.chromium.png|9| Changed|
|Table layout table.single select.chromium.png|9| Changed|
|Table layout table.size - extra small.chromium.png|7| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|6| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|19| Changed|
|Table layout table.size - medium - RTL.chromium.png|9| Changed|
|Table layout table.size - medium.chromium.png|9| Changed|
|Table layout table.size - small.chromium.png|7| Changed|
|Table table - subtle selection.rest.chromium.png|9| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|9| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>